Objective-C/CBLReplicator.mm

```diff
@@ -276,7 +274,7 @@ - (bool) _setupC4Replicator: (C4Error*)outErr {
             .pull = mkmode(isPull(_config.replicatorType), _config.continuous),
             .pushFilter = filter(_config.pushFilter, true),
             .pullFilter = filter(_config.pullFilter, false),
-            .callbackContext    = (__bridge void*)self,
+            .callbackContext    = (__bridge void*)col,
```

Comment:
As you are using collection map, I think you can just set it to self. Also, once LiteCore starts to send back this context, you will not be able to get to the replicator object to call the filter function.